.apko/.bazelrc

```diff
@@ -0,0 +1,6 @@
+# Generated by apko_bazelrc. DO NOT EDIT
+# Required for range requests for fetching the apk packages.
+# https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests
+# Use https://www.base64decode.org/ to see what this base64 encoded command does.
```

Comment:
Would be great to have some public location talking about the problem and this clever solution, maybe a blog.aspect.dev post to start with? Or something on docs.aspect.build?